tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -1322,8 +1322,7 @@ void libfmt_formatter_test_runtime_width() {
     assert(format(STR("{0:{1}}"), 42ull, 7) == STR("     42"));
     assert(format(STR("{0:{1}}"), -1.23, 8) == STR("   -1.23"));
     assert(format(STR("{0:{1}}"), -1.23l, 9) == STR("    -1.23"));
-    assert(format(STR("{0:{1}}"), reinterpret_cast<void*>(0xcafe), 10)
-           == STR("0xcafe    ")); // behavior differs from libfmt, but conforms

```

Comment:
Can you comment on libfmt behavior?

Reply:
Yes, libfmt aligns right: https://gcc.godbolt.org/z/xx466r8dd